
#IndexError
my_list = [10, 20, 30]
print(my_list[3])


#KeyError
my_dict = {'name': 'John', 'age': 30}
print(my_dict['gender'])

